### PR TITLE
Expose MSAA option for libretro.

### PR DESCRIPTION
**Pull request by stuken**
Tested locally and works on Windows/Nvidia and safely updates at runtime.

### CODE REVIEW

**Review by hrydgard — COMMENTED** [review]
I guess there's no way in libretro to only make it appear in Vulkan mode?

File: libretro/libretro_core_options.h [hunk 1]
```diff
@@ -283,6 +283,22 @@ struct retro_core_option_v2_definition option_defs_us[] = {
       },
       "480x272"
    },
+   {
+      "ppsspp_mulitsample_level",
+      "MSAA Antialising (Vulkan Only)",
```
[the remark this answers is not in the document]
Comment (stuken) [reply libretro/libretro_core_options.h]: Fixed.

**Comment by stuken** [comment]
Doesnt look like it other than via the hw render context but thats a chicken egg problem as I belive the options are parsed before the hw context inits and since the frontend can have a renderer override so it doesnt provide that to a core as the launching frontend hw context may not match the potential override.

**Comment by hrydgard** [comment]
That's alright then.

**Review by hrydgard — APPROVED** [review]

**Pull request merged**